### PR TITLE
Fix #334

### LINKED ISSUE #334: It isn't possible to change an existing router node to an action, save.
This throws the `Cannot find action...` error in `store/helpers.ts`' `getActionIndex` method.

### PR DESCRIPTION
**Pull request by ycleptkellan**
- Adds a mutator to replace a router node w/ an action, à la `spliceInRouter`. 
- Resolves #334.

### CODE REVIEW

**Comment by ycleptkellan** [comment]
ready for ya @ericnewcomer

File: src/store/__snapshots__/thunks.test.ts.snap [hunk 1]
```diff
@@ -1,443 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Flow Manipulation init should fetch and initalize flow 1`] = `
```
Comment (ycleptkellan) [inline src/store/__snapshots__/thunks.test.ts.snap]: not quite sure what's going on here

File: src/store/mutators.ts [hunk 2]
```diff
@@ -161,6 +161,31 @@ export const updateAction = (nodes: RenderNodeMap, nodeUUID: string, action: Any
     });
 };
 
+export const spliceInAction = (
```
Comment (ericnewcomer) [inline src/store/mutators.ts]: feel like this closer to `convertToAction(..)`

File: src/store/mutators.ts [hunk 3]
```diff
@@ -161,6 +161,31 @@ export const updateAction = (nodes: RenderNodeMap, nodeUUID: string, action: Any
...
+): RenderNodeMap => {
+    const { [nodeUUID]: originalRenderNode } = nodes;
+    return mergeNode(
+        nodes,
+        mutate(originalRenderNode, {
+            node: {
+                // Append action to node
+                actions: { $set: [action] },
+                // Off any exit but the first, remove the first's name
+                exits: { $set: [{ ...originalRenderNode.node.exits[0], name: null }] },
```
Comment (ericnewcomer) [inline src/store/mutators.ts]: _Other_ exit instead like we discussed

File: src/store/mutators.ts (hunk 2 above)
Comment (ycleptkellan) [reply src/store/mutators.ts]: It's merging it into the `RenderNodeMap` as well though. Should this method only be concerned w/ converting the node to an action?

**Pull request merged**